install

```diff
@@ -1,21 +1,49 @@
 #!/bin/bash
 
+# Safe apt-get that waits until all other instances
+# of apt-get have completed (and released the lock)
+# before executing. This is to prevent the script
+# from failing due to a dpkg database lock.
+#
+# Takes the same arguments as apt-get
+safe-apt-get() {
+    i=0
+    
+    # Wait while the dpkg lock is set
+    while fuser /var/lib/dpkg/lock >/dev/null 2>&1 ; do
+        case $(($i % 4)) in
+            0 ) j="-" ;;
+            1 ) j="\\" ;;
+            2 ) j="|" ;;
+            3 ) j="/" ;;
+        esac
+        echo -en "\r[$j] Waiting for other software managers to finish..." 
+        sleep 0.5
+        ((i=i+1))
+    done 
+
+    sudo apt-get "$@"
+}
+
 # Makes this script exit on first error.
 set -e
 
-# need this to create venv if python was installed without the entire stdlib
-sudo apt-get -y install python3.4-venv
-
-# need this to be able to install ds4drv
-sudo apt-get -y install python3-dev
+# need this to be able to install ds4drv, and this should be done outside the
+# virtualenvironment
+safe-apt-get -y install python-dev
 
-# create a python3 virtualenvironment in the buzzmobile directory
-# yes, venv will allow you to create a virtualenv in any directory
-python3 -m venv .
+# create a python2 virtualenv in the buzzmobile directory using python3
+# yes this is intentional, we're doing this because virtualenv and venv
+# function differently, and venv isn't availible on python2
+safe-apt-get -y install python-virtualenv
+python -m virtualenv .
 
 # activate the virtualenv we just created
 source bin/activate
 
+# remove a name conflict with catkin_make later
+rm local/buzzmobile
```

Comment:
check the commit where I did this, essentially if you have `venv`, `venv/local` is a folder that has symlinks to everything in `venv` (because python2 import path issues I think?). This is problematic if buzzmobile is there, because its a symlink to the buzzmobile rospackage, which will confuse `catkin_build`. see https://github.com/gtagency/buzzmobile/pull/155/commits/08da3921230fbaff183c34fdaadf3d1112fbac10